builtin/nomad/task.go

```diff
@@ -263,9 +263,9 @@ func (p *TaskLauncher) StartTask(
 
 	// On-Demand runner specific configuration to start the task with
 	config := map[string]interface{}{
-		"image":   tli.OciUrl,
-		"args":    tli.Arguments,
-		"command": tli.Entrypoint,
+		"image":      tli.OciUrl,
+		"args":       tli.Arguments,
+		"entrypoint": tli.Entrypoint,
```

Comment:
Interesting, surprised we didn't catch this with other ODR jobs! Do we not need to also set the command for the launched resource too?

Reply:
Nope! Setting the command in addition to the entrypoint here would just add args to the command that the entrypoint runs.